pkgs/development/python-modules/aiosonic/default.nix

```diff
@@ -0,0 +1,59 @@
+{
+  pkgs,
+  lib,
+  buildPythonPackage,
+  pythonOlder,
+  fetchFromGitHub,
+  poetry-core,
+  # install_requires
+  charset-normalizer,
+  h2,
+  onecache,
+  pytest,
+}:
+
+buildPythonPackage rec {
+  pname = "aiosonic";
+  version = "0.20.1";
+  pyproject = true;
+
+  disabled = pythonOlder "3.8";
+  doCheck = true;
+
+  src = fetchFromGitHub {
+    owner = "sonic182";
+    repo = "aiosonic";
+    rev = "refs/tags/${version}";
+    hash = "sha256-RMkmmXUqzt9Nsx8N+f9Xdbgjt1nd5NuJHs9dzarx8IY=";
+  };
+
+  __darwinAllowLocalNetworking = true;
```

Comment:
```suggestion
```
I don't think we need this

Reply:
That was supposed to activate local networking for the tests. Could I get some clarification from you on why this is not allowed?